distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,368 @@
+====
++
+[source, bash, linenums]
+----
+chown -R NON_ROOT_USER ${branding}_HOME ${branding}_HOME/etc ${branding}_HOME/data
+chgrp -R NON_ROOT_USER ${branding}_HOME/etc ${branding}_HOME/data
+chmod -R og-w ${branding}_HOME/etc ${branding}_HOME/data
+----
++
+. Restrict access to sensitive files by ensuring that the only users with “group” permissions (e.g., ${branding-lowercase}-group) have access.
+. Execute the following command on the above files (examples assume `${branding}_HOME` is `<${branding}_HOME>`):
++
+[source,bash]
+----
+chmod -R o <${branding}_HOME>
+----
++
+. As the application owner (e.g., ${branding-lowercase} user), restrict access to sensitive files.
++
+[source,bash,linenums]
+----
+chmod 640 <${branding}_HOME>/etc
+chmod 640 <${branding}_HOME>/deploy
+----
+////
+// New Version
+* Change ownership of ${branding}_HOME
+** `chown -R NON_ROOT_USER ${branding}_HOME`
+* Change group ownership on sub-directories
+** `chgrp -R ${branding}_GROUP ${branding}_HOME/etc ${branding}_HOME/data ${branding}_HOME/deploy`
+* Change group permissions
+** `chmod -R g-w ${branding}_HOME/etc ${branding}_HOME/data ${branding}_HOME/deploy`
+* Remove permissions for users
+** `chmod -R o-rwx ${branding}_HOME/etc ${branding}_HOME/data ${branding}_HOME/deploy`
+
+
+[IMPORTANT]
+====
+The system administrator must restrict certain directories to ensure that the application (user) cannot access restricted directories on the system.
+For example the `NON_ROOT_USER` should only have read access to `<${branding}_HOME.
+====
+
+==== Java Requirements
+
+* Install/Upgrade to Java 8 http://www.oracle.com/technetwork/java/javase/downloads/index.html[J2SE 8 SDK]
+** The recommended version is http://www.oracle.com/technetwork/java/javase/8u60-relnotes-2620227.html[8u60] or later.
+* Supported platforms are *NIX - Unix/Linux/OSX, Solaris, and Windows.
+* http://www.oracle.com/technetwork/java/javase/downloads/index.html[JDK8] must be installed.
+* The `JAVA_HOME` environment variable must be set to the location where the JDK is installed.
+
+.Setting `JAVA_HOME` on *NIX
+----
+JAVA_HOME=/usr/java/jdk1.8.0
+export JAVA_HOME
+----
+
+.Setting `JAVA_HOME` on Windows
+----
+set JAVA_HOME=C:\Program Files\Java\jdk1.8.0
+----
+
+.*NIX
+[WARNING]
+====
+Unlink `/usr/bin/java` if it is already linked to a previous version of the JRE:
+`unlink /usr/bin/java`
+====
+
+* Verify that the `JAVA_HOME` was set correctly.
+
+.*NIX
+----
+echo $JAVA_HOME
+----
+
+.Windows
+----
+echo %JAVA_HOME%
+----
+
+* ${branding} installation zip file.
+* A web browser.
+* For Linux systems, increase the file descriptor limit by editing `/etc/sysctl.conf` to include or change the following:
+
+.File Descriptor Limit
+----
+fs.file-max = 6815744
+----
+
+.Restart
+[WARNING]
+====
+For the change to take effect, a restart is required.
+====
+
+. *Nix Restart Command
+----
+init 6
+----
+
+==== Deployment Guidelines
```

Comment:
Like section 2.1.1, this section feels out of place. Should we move it in the same section as 2.1.1, e.g., as a sub-section of 2.2?